scripts/ubuntu1604/lineage.sh

```diff
@@ -502,70 +470,44 @@ source build/envsetup.sh
 sed -i -e "s/-Xmx2048m/-Xmx4096m/g" \$HOME/android/lineage/build/tools/releasetools/common.py
 
 # Download and configure the environment for the device.
-breakfast \$DEVICE
-
-# # Find the latest upstream build.
-# ARCHIVE=\`curl  --location --silent https://download.lineageos.org/\$DEVICE | grep href | grep https://mirrorbits.lineageos.org/full/\$DEVICE/ | head -1 | awk -F'"' '{print \$2}'\`
-#
-# # Create a system dump directory.
-# mkdir -p \$HOME/android/system_dump/ && cd \$HOME/android/system_dump/
-#
-# # Download the archive.
-# curl --location --output lineage-archive.zip "\$ARCHIVE"
-#
-# # Extract the system blocks.
-# unzip lineage-archive.zip system.transfer.list system.new.dat
-#
-# # Clone the sdat2img tool.
-# git clone https://github.com/xpirt/sdat2img
-#
-# # Convert the system block file into an image.
-# python sdat2img/sdat2img.py system.transfer.list system.new.dat system.img
-#
-# # Mount the system image.
-# mkdir -p \$HOME/android/system/
-# sudo mount system.img \$HOME/android/system/
-
-# Change to the device directory and run the extraction script.
-# cd \$HOME/android/lineage/device/\$VENDOR/\$DEVICE
-# ./extract-files.sh \$HOME/android/system_dump/system
-#
-# # Unmount the system dump.
-# sudo umount \$HOME/android/system_dump/system
-# rm -rf \$HOME/android/system_dump/
-
-# Extract the vendor specific blobs.
-mkdir -p \$HOME/android/vendor/system/
-tar -xzv -C \$HOME/android/vendor/ -f \$HOME/system-blobs.tar.gz
-
-# Change to the device directory and run the extraction script.
-cd \$HOME/android/lineage/device/\$VENDOR/\$DEVICE
-./extract-files.sh \$HOME/android/vendor/system/
-
-# Cleanup the vendor blob files.
-rm -rf \$HOME/android/vendor/
-
-# Setup the environment.
-cd \$HOME/android/lineage/ && source build/envsetup.sh
 breakfast \$DEVICE || ( printf "\n\n\nBuild failed. (breakfast)\n\n\n"; exit 1 )
 
 # Setup the cache.
 cd \$HOME/android/lineage/
-prebuilts/misc/linux-x86/ccache/ccache -M 20G
+
+export CCACHE_EXEC="prebuilts/misc/linux-x86/ccache/ccache"
+
+if [ ! -x "\$CCACHE_EXEC" ]; then
+  export CCACHE_EXEC="\$(which ccache)"
+fi
+
+"\$CCACHE_EXEC" -M 20G
+
+BUILDSTAMP=\`date --utc +'%Y%m%d'\`
```

Comment:
[SC2006](https://github.com/koalaman/shellcheck/wiki/SC2006) - 
```suggestion
BUILDSTAMP=\$(date --utc +'%Y%m%d')
```
This syntax change could be made many other places in this codebase.